### ISSUE TITLE
Redirect to https

### ISSUE THREAD

**Issue by mcdemarco** [mcdemarco/conclar#7]
The current rewrite rules are for React router, so http is again accessible.  This isn't as bad as with the chrome cache manifest bug in konopas, but it could still lead to confusion about where people's My Con settings have gone if they switch protocol unknowingly.

**Comment by lostcarpark** [comment]
This is a good point. I think having redirect to HTTPS in the web configuration is good practice, but shouldn't be imposed on sites, as some (foolish) people still only have HTTP sites. I consider the included .htaccess a starting point showing how to direct requests to index.html for React routing.

I think putting suggestions in the Readme on how to implement https redirection would be a good idea, but I'm not sure they should go in the default file included.

**Comment by mcdemarco** [comment]
Yeah, I wouldn't force it on people, but I haven't figured it out yet by googling so if/when we do we should definitely document it.

**Comment by mcdemarco** [comment]
We did this this year, so I just need to document it:

```
<IfModule mod_rewrite.c>
    RewriteEngine On
    RewriteCond %{HTTPS} off
    RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI} [L]
    RewriteBase /
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_FILENAME} !-l
    RewriteRule . /index.html [L]
</IfModule>
```

Also note that it's preferable to do https redirection in the Apache config instead, if you can.
